tests/fixtures/bacterial_example/intermediates/fastANI/all_vs_NC_002696.fna.fastani

```diff
@@ -0,0 +1,4 @@
+../fixtures/bacterial_example/NC_002696.fna.gz	../fixtures/bacterial_example/NC_002696.fna.gz	100	1338	1338
+../fixtures/bacterial_example/NC_010338.fna.gz	../fixtures/bacterial_example/NC_002696.fna.gz	82.9124	877	1825
+../fixtures/bacterial_example/NC_011916.fna.gz	../fixtures/bacterial_example/NC_002696.fna.gz	99.9946	1337	1347
+../fixtures/bacterial_example/NC_014100.fna.gz	../fixtures/bacterial_example/NC_002696.fna.gz	86.1137	998	1551
```

Comment:
I noticed there the FASTA extension has been included in the filename - a quirk of the ``pathlib.Path`` and how it handles ``Path("example.fasta.gz").stem`` as ``example.fasta``

Not a big problem for what are essentially just internal temp files, but a quirk we might want to address?